test/functional/inspec_exec_test.rb

```diff
@@ -1085,6 +1085,34 @@ def stderr
     end
   end
 
+  describe "When specifying a config file and --reporter option to configure reporter in a run in a correct manner" do
+    it "should obey the configurations of both cli reporter and config reporter options" do
+      outpath = Dir.tmpdir
+      cli_args = "--no-create-lockfile --reporter json:#{outpath}/foo/bar/test.json --config " + File.join(config_dir_path, "json-config", "reporter-json-config.json")
+      inspec("exec #{complete_profile} #{cli_args}")
+
+      # File specified with cli reporter option - test to see file exists
+      _(File.exist?("#{outpath}/foo/bar/test.json")).must_equal true
+      _(File.stat("#{outpath}/foo/bar/test.json").size).must_be :>, 0
+      # STDOUT true set using config - test to see if this is obeyed
+      _(stdout).must_include "complete example profile (complete)"
+      _(stdout).must_include "1 successful control"
+      _(stdout).must_include "0 control failures"
+      _(stdout).must_include "0 controls skipped"
+      _(stderr).must_be_empty
+      assert_exit_code 0, out
+    end
+  end
+
+  describe "When specifying a config file and --reporter option to configure reporter with stdout true from both the options" do
+    let(:cli_args) { "--config " + File.join(config_dir_path, "json-config", "reporter-json-config.json") + " --reporter json html2" }
+    let(:run_result) { run_inspec_process("exec " + File.join(profile_path, "basic_profile") + " " + cli_args) }
+    it "should raise error that only single reporter can have output to stdout" do
```

Comment:
Instead of raising an error can we show the warning and give precedence to any of them depending that way we won't be breaking the flow of generating a report.

Reply:
This is the current way of how if you add multiple reporter which have `stdout: true`, then it give user this message to change the way they are using reporter.
This happens in all the scenarios:
1. While using `--reporter cli json` - No file mentioned, but multiple reporters to output
2. While using `stdout: true` in config file, for multiple reporter options
3. While using `stdout: true` in config file in reporter options and also using `--reporter REPORTER_OPTION_WITHOUT_FILE`

So this is the current behaviour. Not bringing a new change with this implementation.